### PR TITLE
Format the vision input from server

### PR DESCRIPTION
**Pull request by ioarun**
I have formatted the vision input from server so that now it is similar to the vision input in the openai gym environments like ATARI games.

### CODE REVIEW

File: gym_torcs.py [hunk 1]
```diff
@@ -226,15 +226,18 @@ def agent_to_torcs(self, u):
...
-        r = np.array(r).reshape(sz)
-        g = np.array(g).reshape(sz)
-        b = np.array(b).reshape(sz)
-        return np.array([r, g, b], dtype=np.uint8)
+        rgb = []
+        temp = []
+        # convert size 64x64x3 = 12288 to 64x64=4096 2-D list 
+        # with rgb values grouped together.
+        # Format similar to the observation in openai gym
+        for i in range(0,12285,3):
```
Comment (ugo-nama-kun) [inline gym_torcs.py:234]: This range should be (0, 12286, 3). This caused a bug in the example code and I fixed it manually. Thanks!

**Comment by ugo-nama-kun** [comment]
Thank you for your contribution!

File: gym_torcs.py (hunk 1 above)
Comment (ioarun) [reply gym_torcs.py:234]: My bad! Thanks for correcting.

**Comment by dasGringuen** [comment]
Will this affect performance? It's iterating with Python

**Pull request merged**